docs/webhooks.rst

```diff
@@ -40,6 +40,17 @@ your docs whenever you push updates:
 If you ever need to manually set the webhook on Bitbucket,
 you can point it at ``https://readthedocs.org/bitbucket``.
 
+GitLab
+------
+
+If your project is hosted on GitLab, you can manually set the webhook on Gitlab and
+point it at ``https://readthedocs.org/gitlab``:
```

Comment:
So, without a webhook management page, these webhooks aren't completely deprecated yet. Once that feature exists, this page will need to be updated to instead reference that page and drop information on manual set up. Likewise, we should add a deprecation notice on GitHub Services for RTD as well. I'll open an issue outlining the work that will go into this one -- fine to point to this webhook endpoint for now.